src/mflux/models/qwen/variants/edit/qwen_image_edit.py

```
@@ -0,0 +1,263 @@
+        Callbacks.before_loop(
+            seed=seed,
+            prompt=prompt,
+            latents=latents,
+            config=runtime_config,
+        )
+
+        for t in time_steps:
+            try:
+                # 4.t Concatenate the updated latents with the static image latents
+                hidden_states = mx.concatenate([latents, static_image_latents], axis=1)
+                hidden_states_neg = mx.concatenate([latents, static_image_latents], axis=1)  # noqa: F841
```

Ruff (0.14.4)

Unused `noqa` directive (unused: `F841`)

Remove unused `noqa` directive

(RUF100)